cli/lib/kontena/plugin_manager/common.rb

```diff
@@ -26,7 +26,7 @@ def use_dummy_ui
 
       # Prefix a plugin name into a gem name (hello to kontena-plugin-hello)
       def prefix(plugin_name)
-        return plugin_name if plugin_name.to_s.start_with?('kontena-plugin-')
+        return plugin_name if plugin_name.to_s.start_with?('kontena-plugin-') || plugin_name.include?('.')
```

Comment:
Somehow broken with `kontena plugin search` => `search_plugins(nil) => `prefix(nil)`, although not sure how it's worker earlier either?

```
$ DEBUG=true kontena plugin search 
DEBUG Kontena CLI 1.5.0.dev (ruby-2.3.1+x86_64-linux-gnu)
DEBUG Running Kontena::MainCommand with ["plugin", "search"] -- callback matcher = 'nil'
DEBUG Running Kontena::Cli::PluginCommand with ["search"] -- callback matcher = 'nil'
DEBUG Running Kontena::Cli::Plugins::SearchCommand with [] -- callback matcher = 'plugins search'
DEBUG Execution took 0.121 seconds
DEBUG NoMethodError
/home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/plugin_manager/common.rb:29:in `prefix': undefined method `include?' for nil:NilClass (NoMethodError)
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/plugin_manager/common.rb:69:in `search_plugins'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/cli/plugins/search_command.rb:13:in `execute'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/command.rb:215:in `run'
	from /home/terom/.gem/ruby/2.3.0/gems/clamp-1.1.2/lib/clamp/subcommand/execution.rb:11:in `execute'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/command.rb:215:in `run'
	from /home/terom/.gem/ruby/2.3.0/gems/clamp-1.1.2/lib/clamp/subcommand/execution.rb:11:in `execute'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/command.rb:215:in `run'
	from /home/terom/.gem/ruby/2.3.0/gems/clamp-1.1.2/lib/clamp/command.rb:132:in `run'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/bin/kontena:19:in `<top (required)>'
	from /home/terom/.local/bin/kontena:23:in `load'
	from /home/terom/.local/bin/kontena:23:in `<main>'
```

Reply:
Wouldn't hurt to fix the `plugin search` spec to use `run!` as well, while you're at it

```
Failures:

  1) plugin search lists available plugins
     Failure/Error: expect(k.code).to eq(0)
     
       expected: 0
            got: 1
     
       (compared using ==)
     # ./spec/features/plugin/search_spec.rb:6:in `block (2 levels) in <top (required)>'

```